action.yml

```diff
@@ -52,5 +52,5 @@ outputs:
   skipped:
     description: Will be set to true if skip-on-empty is enabled and a release was not created, false otherwise
 runs:
-  using: 'node16'
+  using: 'node12'
```

Comment:
?

Reply:
`node16` throws an error. Seems like it is not supported by Github Actions:
(I loaded lastet version of this repo – `fregante/release-with-changelog@main`)
```
Error: System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: node16' is not supported, use 'docker' or 'node12' instead.')
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Fail to load fregante/release-with-changelog/main/action.yml
```